tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -130,42 +142,178 @@ private static ArgumentParser argParser() {
                 .metavar("THROUGHPUT")
                 .help("throttle maximum message throughput to *approximately* THROUGHPUT messages/sec");
 
+        parser.addArgument("--num-threads")
+                .action(store())
+                .required(true)
+                .type(Integer.class)
+                .metavar("NUM-THREADS")
+                .dest("numThreads")
+                .help("The value bound of the random integers in message payload to simulate different compression ratio.");
+
+        parser.addArgument("--value-bound")
+                .action(store())
+                .required(true)
```

Comment:
Would it be better to provide default value for this configuration? Otherwise this patch will break existing tests/tools that use `ProducerPerformance`.